canu-docker

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env sh
```

Comment:
Instead of docker compose as sits in the repo currently, please look at a wrapper script like https://github.com/Cray-HPE/canu/blob/devel-container/canu-prod.  The `canu-prod` and `canu-devel` wrappers are what were aimed at replacing the pyinstaller binary.  Focus here would be on the "production" wrapper.  

The wrapper in the devel-container branch worked with podman and docker and was tested on Mac OSX + Docker, SLES + podman and Fedora + podman.  Use a `canu-prod` style wrapper versus the docker-compose.  Users shouldn't have to know or interact w/ Docker or Podman for most use cases.

Reply:
Wilco.

Reply:
Fixed up.

```
% ./canu-docker                                                                                                           6:50

./canu-docker [devel|prod] CONTAINER_NAME [rebuild|same] [args]
```

I combined the two scripts you had into one since they were both essentially the same code, with just a few different flags/options.

Reply:
@trad511 does that work for you?